actionpack/test/controller/log_subscriber_test.rb

```diff
@@ -258,6 +260,18 @@ def test_with_fragment_cache
     @controller.config.perform_caching = true
   end
 
+  def test_with_fragment_cache_when_log_disabled
+    @controller.config.perform_caching = true
+    ActionController::Base.enable_fragment_cache_logging = false
+    get :with_fragment_cache
+    wait
+
+    assert_equal 2, logs.size
```

Comment:
We should also add some `assert_match` calls, only checking the size doesn't exactly boost my confidence that things are working.